ajax/libs/hls.js/package.json

```diff
@@ -2,11 +2,11 @@
   "name": "hls.js",
   "version": "0.10.1",
```

Comment:
Should I update the `version` even if `autoupdate` is specified ?

Reply:
no need